slicetest/run.go

```diff
@@ -10,36 +10,45 @@ package slicetest
 import (
 	"context"
 	"reflect"
+	"sync"
 	"testing"
 
+	"github.com/grailbio/bigmachine"
 	"github.com/grailbio/bigslice"
 	"github.com/grailbio/bigslice/exec"
 	"github.com/grailbio/bigslice/sliceio"
 )
 
-// Run evaluates the provided slice in local execution mode,
-// returning a scanner for the result. Errors are reported as fatal
-// to the provided t instance. Run is intended for unit testing of
-// Slice implementations.
-func Run(t *testing.T, slice bigslice.Slice) *sliceio.Scanner {
+var sessOnce sync.Once
+var sess *exec.Session
```

Comment:
Optional: I like using `var` blocks to indicate that once, mu, etc. protect variables:
```
var (
  sessOnce sync.Once
  sess *exec.Session
)
```